### PR TITLE
Add new audit log events

### PR DESCRIPTION
**Pull request by androse**
- Adds workspace_default_session_duration_changed,  workspace_announcement_created, workspace_announcement_removed events
- Reformats the Admin Settings section to include Resource Type
- Renames AdvancedSearchQuery resource type to Search to match the name used in the API

### CODE REVIEW

File: source/includes/markdown/_17-audit-log.html.md [hunk 1]
```diff
@@ -38,26 +38,27 @@ All user events operate on the _User_ resource type.
...
-| workspace_default_team_privacy_settings_changed  | The workspace's default team privacy settings were changed. |
-| workspace_wide_reporting_enabled                 | Workspace wide reporting was enabled. |
-| workspace_wide_reporting_disabled                | Workspace wide reporting was disabled. |
-| workspace_associated_email_domain_added          | An email domain was added to the workspace. |
-| workspace_associated_email_domain_removed        | An email domain was removed from the workspace. |
-| workspace_require_two_factor_auth_enabled        | Two factor authentication was set as required for the workspace. |
-| workspace_require_two_factor_auth_disabled       | Two factor authentication was set as not required for the workspace. |
-| workspace_view_links_enabled                     | Read-only link sharing was enabled for the workspace. |
-| workspace_view_links_disabled                    | Read-only link sharing was disabled for the workspace. |
+| Event Type | Resource Type | Description |
```
Comment (aw-asana) [inline source/includes/markdown/_17-audit-log.html.md:41]: @androse , do you mind switching this back to a two-column table? With the way that our docs are designed, the changes look a bit cramped:

Screen Shot 2022-01-21 at 4 10 17 PM

File: source/includes/markdown/_17-audit-log.html.md [hunk 2]
```diff
@@ -38,26 +38,27 @@ All user events operate on the _User_ resource type.
  
 ### Admin Settings
  
-All admin settings events operate on the _Workspace_ resource type.
```
Comment (aw-asana) [inline source/includes/markdown/_17-audit-log.html.md:41]: @androse  If we keep the table at just two columns, maybe we can re-include this, and just note that `workspace_announcement_created` and `workspace_announcement_removed` operate on different resources as well.

Comment (androse) [reply source/includes/markdown/_17-audit-log.html.md:41]: Sounds like a reasonable compromise.

**Review by aw-asana — APPROVED** [review]
LGTM. Thanks for making the changes @androse

**Pull request merged**